Add react player

samuel-skean opened the pull request:
Attempts to resolve issue #7.
Depends on the ReactPlayer module, uses it to create a very simple VideoPlayer component with a form that lets the user specify a URL.

samuel-skean committed:
Added React-Player as a dependency.

samuel-skean committed:
Made a dummy VideoPlayer component that automatically plays the video when a loadable URL is supplied in its input field.

It looks disgusting and it doesn't really report errors or anything, but
it works.

samuel-skean committed:
Video URL only updates on form submission.

samuel-skean committed:
Removed superfluous hi

samuel-skean committed:
Removed superfluous ReactPlayer component.

samuel-skean committed:
Merge branch 'main' into add-react-player

Ajknight121 commented on src/components/VideoPlayer.jsx:
```diff
@@ -0,0 +1,28 @@
+import ReactPlayer from 'react-player';
+
+export const VideoPlayer = () => {
+    // Highly informed by this w3schools tutorial:
```
They are the goat

Ajknight121 commented on src/components/VideoPlayer.jsx:
```diff
@@ -0,0 +1,28 @@
+        setPlayerUrl(inputUrl);
+    }
+
+    return <>
```
Please surround return in parenthesis

Ajknight121 reviewed (changes requested):
Awesome work! Just to wrap this up can you center the video player on the screen and I'll pull this in.

samuel-skean committed:
Fixed formatting.

Added parentheses around return value.
Changed indentation to 2 spaces to match rest of project.

samuel-skean committed:
Centered the video player.

Also, formatted with prettier.js.

Ajknight121 reviewed (approved):
Good work!
